lametro/models.py

```diff
@@ -153,6 +153,56 @@ def board_report(self):
 
         return br
 
+    @property
+    def actions_and_agendas(self):
+        '''
+        Return list of dictionaries, each representing an action or an agenda on
+        which the given bill appears, in the format:
+
+        {
+            'date': <datetime>,
+            'description': <str>,
+            'event': <LAMetroEvent>,
+            'organization': <Organization>
+        }
+        '''
+        actions = self.actions.all()
+        events = LAMetroEvent.objects.filter(agenda__related_entities__bill=self)
+
+        data = []
+
+        for action in actions:
+            event = LAMetroEvent.objects.filter(\
+                participants__entity_type='organization',\
+                participants__name=action.organization,
+                start_time__date=action.date)
+
+            
+            action_dict = {
+                'date': action.date_dt.date(),
```

Comment:
`date_dt` changed in https://github.com/datamade/django-councilmatic/pull/272 -- it now returns a date object! (You may have to rebase your branch off master, then rebuild the app image with `docker-compose build app` to capture the upgraded django-councilmatic dependency to test this locally.)